scripts/test.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash -e
 
-pub run test test/
+flutter test test/
```

Comment:
I don't think the directory name is needed

Reply:
let me try

Reply:
of course I want to replace this with our standard scripts later